src/infra/storage/storage.service.spec.ts

```diff
@@ -42,32 +42,56 @@ describe('StorageService', () => {
 	});
 
 	describe('onModuleInit', () => {
-		describe('when bucket exists', () => {
+		describe('when makeBucket resolves', () => {
+			it('should call makeBucket', async () => {
```

Comment:
Ich bin mir gar nicht sicher ob der Test Code hier so stehen sollte. Die onInit und destroy Methoden haben was mit der Initalisierung zu tun. Sicher Code von uns geschrieben. Aber gehört das nicht trotzdem eher in das beforeAll hinter 
service = moduleFixture.get(StorageService); ?

Momentan ist das ein nicht isolierter Test. Damit onModuleInit richtigerweise ausgeführt wurde, muss ich den Test als erstes machen, ansonsten wurde es nicht für die anderen vorher ausgeführt. Klinkt falsch. 

Wenn wir den Code seperate testen wollen, bräuchten wir ein weiteren describe mit eigenem beforeAll + module zusammen bauen um die Isolierung zu wahren.